### ISSUE TITLE
Increase session time out time

### ISSUE THREAD

**Issue by jozefbakus** [AbsaOSS/hyperdrive-trigger#239]
@zakera786  
While working with creating workflows , the session times out however i am only aware of this once I try to save the workflow. When saving workflow the screen re-directs me to login page. I will try and time myself so i can give correct feedback on this.
UI timed out within 10 minutes.

Increase session time out to 1 hour.

**Comment by kevinwallimann** [comment]
I couldn't reproduce a session timeout within 10 minutes on DEV. I tried 10 mins, 20 mins, 28 mins and 32 mins. The only timeout I got was at 32 mins, which is as expected since the default session timeout is 30 mins. https://github.com/spring-projects/spring-boot/blob/97a71be9b8e2cf4b2fe0625896975865d6a8fdab/spring-boot-project/spring-boot/src/main/java/org/springframework/boot/web/servlet/server/Session.java#L38

With spring boot, we can set the timeout to 60 mins with the following parameter.
`server.servlet.session.timeout=60m`

With the external tomcat, it can be set in {tomcat-root}/conf/web.xml
```
    <session-config>
        <session-timeout>60</session-timeout>
    </session-config>
```
